.github/workflows/dependabot-batcher.yml

```
@@ -0,0 +1,38 @@
+# Dependabot Batcher v1.0.1
+
+name: 'Dependabot Batcher'
+
+# Controls when the workflow will run
+on:
+  push:
+    branches: [ "r/12.x", "develop" ]
+  pull_request:
+    branches: [ "r/12.x", "develop" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  dependabot-batcher:
+    name: 'Combine dependabot PRs'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repo'
+        uses: actions/checkout@v3
+        with:
+          token: ${{ secrets.DEPENDABOT_BATCHER }}
+
+      - name: 'Set Node Version'
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18.8.0
+
+  # Batches Dependabot PRs into one by merging them into a combined branch, then raising a new PR
+      - name: 'Dependabot Batcher'
+        if: ${{ contains(github.head_ref, '/modules/engage-paella-player-7') }}
+        uses: Legal-and-General/dependabot-batcher@v1.0.1
+        with:
+          token: ${{ secrets.DEPENDABOT_BATCHER }} #required
+          baseBranchName: 'r/12.x' #optional
+          batchBranchName: 'dep-batcher-updates' #optional
+          batchPullTitle: 'Dependabot Batcher Updates' #optional
```

Maybe include Paella, since it's for Paella 7 specifically?

```suggestion
          batchBranchName: 'paella7-batcher-updates'
          batchPullTitle: 'Batch Dependabot Updates for Paella 7'
```

---

I like that change, makes easier to know for what is the action. Since is only for PP 7 makes sense